GOVERNANCE.md

```diff
@@ -0,0 +1,54 @@
+The **WGPU project** is an ecosystem that _enables application authors to write
```

Comment:
Yeah, i would say it _has_ an ecosystem, but is not one itself

Reply:
> Is WGPU not an ecosystem, though?

Well, I would understand “the WGPU ecosystem” as the WGPU project plus those third-party crates. But “third-party” here means, at a minimum, “not governed by the WGPU project organization” — and to me, “ecosystem” further implies that those parties created the crates _by their own choices, to serve their own goals_ and as a consequence made WGPU more useful for everyone.

This document describes governance for the WGPU project, not things that are not the WGPU project. Governing the “ecosystem” by these principles and leadership is a logical contradiction, because if you could, it would cease to be an “ecosystem” and become part of the project.

Reply:
Changed as suggested with 7a73c1622.